src/views/FontWatcher.tsx

```diff
@@ -0,0 +1,61 @@
+import { reaction } from "mobx";
+import { observer } from "mobx-react-lite";
+import React from "react";
+import { Preferences } from "../hooks/stores/preferences";
+
+interface Props {
+  preferences: Preferences;
+}
+
+const DEFAULT_FONTS = {
```

Comment:
See my comment on the else statements for fallbacks; we should define defaults in the client/preferences, and fallback to the underlying stylesheet's base definition (not controlled by us presumably), and not define defaults at all in this file.